src/Parser/Rogue/Assassination/Modules/Legendaries/DuskwalkersFootpads.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import RESOURCE_TYPES from 'common/RESOURCE_TYPES';
+import SpellLink from 'common/SpellLink';
+
+// Equip: The remaining cooldown on Vendetta is reduced by 1 sec for every 65 Energy you expend.
+
+const VENDETTA_CDR_PER_ENERGY = 1/65;
+
+class DuskwalkersFootpads extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+      };
+
+  totalReduction = 0;
+  wastedReduction = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasFeet(ITEMS.DUSKWALKERS_FOOTPADS.id);
+  }
+
+  on_byPlayer_spendresource(event) {
+    if (event.resourceChangeType !== RESOURCE_TYPES.ENERGY.id) {
+      return;
+    }
+
+    const spent = event.resourceChange;
```

Comment:
I think that energy spent on feint does not reduce Vendetta cooldown as of 7.3.5